### PR TITLE
Update description of Viewpoint.follow field 

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Related to https://github.com/omichel/webots/pull/6423.

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
:+1:

**Comment by omichel** [comment]
:eyes:

File: reference/viewpoint.md [hunk 1]
```diff
@@ -43,14 +43,36 @@ The `near`, `far` and the `fieldOfView` fields define together the viewing frust
...
+      DEF TARGET Solid {
+        name "target\solid" // <- followed object
+      }
+    }
+    name "second:solid"
+  }
+  name "robot"
+}
+```
+Then, `follow` value will be "robot:second\:target\\solid".
```
Comment (omichel) [inline reference/viewpoint.md]: That seems wrong, shouldn't it be `"robot:second\:solid:target\\solid"` instead?

Comment (stefaniapedrazzi) [reply reference/viewpoint.md]: Yes, thank you for pointing it out.

**Comment by stefaniapedrazzi** [comment]
:+1:

**Pull request merged**